sdv/data_processing/data_processor.py

```diff
@@ -149,6 +153,59 @@ def get_sdtypes(self, primary_keys=False):
 
         return sdtypes
 
+    @staticmethod
+    def _load_module_from_path(path):
```

Comment:
should we move this to a `utils` file? Seems like it could potentially be reused

Reply:
Had the same idea in mind actually, this confirms to just move it there.